Kiosk/Bid Fulfillment/ConfirmYourBidViewController.swift

```diff
@@ -33,16 +39,27 @@ class ConfirmYourBidViewController: UIViewController {
         let attrTitle = NSAttributedString(string: titleString, attributes:attributes)
         useArtsyLoginButton.setAttributedTitle(attrTitle, forState:useArtsyLoginButton.state)
 
-        RAC(self, "number") <~ numberSignal
+        numberSignal
+            .bindTo(_number)
+            .addDisposableTo(rx_disposeBag)
```

Comment:
Yeah, that's a good idea. I've thought about how to get rid of at least the last line, but it's hard. I either need to pass in the dispose bag or a reference to `self`. Something to keep an eye out for a better abstraction.